saos-webapp/src/main/webapp/WEB-INF/view/common/javaScriptGlobalVariables.jsp

```diff
@@ -11,6 +11,25 @@ var contextPath = '${pageContext.request.contextPath}',
 			judgmentSearchFormFieldLawJournalChoseItem: '<spring:message code="judgmentSearch.formField.lawJournal.choseItem" />',
 			judgmentSearchFormFieldLawJournalNoItems: '<spring:message code="judgmentSearch.formField.lawJournal.noItems" />',
 			
+			contextDateAnyValue: '<spring:message code="context.date.fieldDescription" />: <b><spring:message code="context.date.anyValue" /></b>',
+			
+			to: '<spring:message code="to" />',
+			from: '<spring:message code="from" />',
+			
+			lang: '<spring:message code="page.lang" />',
+			
+			month1: '<spring:message code="month.short.1" />',
+			month2: '<spring:message code="month.short.2" />',
+			month3: '<spring:message code="month.short.3" />',
+			month4: '<spring:message code="month.short.4" />',
+			month5: '<spring:message code="month.short.5" />',
+			month6: '<spring:message code="month.short.6" />',
+			month7: '<spring:message code="month.short.7" />',
+			month8: '<spring:message code="month.short.8" />',
+			month9: '<spring:message code="month.short.9" />',
+			month10: '<spring:message code="month.short.10" />',
+			month11: '<spring:message code="month.short.11" />',
+			month12: '<spring:message code="month.short.12" />',
```

Comment:
in my opinion array here would be more elegant:
`months: ['<spring:message code="month.short.1" />', '<spring:message code="month.short.2" />', ... ],`